### PR TITLE
SKPutRequest  not updating values and untracked RequestId on Put

### PR DESCRIPTION
**Pull request by JONA-GA**
Hello!!
my first PR on Github...
When a PutRequest is received by SensESP , a debuglog message " Untracked RequestID" and the value changed is not updated.
The code presented in PR is working fine for my application based on SmartSwitchcontroller.

### CODE REVIEW

File: src/sensesp/net/ws_client.cpp [hunk 1]
```diff
@@ -274,7 +275,7 @@ void WSClient::process_received_updates() {
   SKListener::take_semaphore();
 
   const std::vector<SKListener*>& listeners = SKListener::get_listeners();
-
+  const std::vector<SKPutListener*>& Plisteners = SKPutListener::get_listeners();
```
Comment (mairas) [inline src/sensesp/net/ws_client.cpp:278]: Nitpick: SensESP is following the Google C++ Style Guide and as such, member variables should have snake_case. So, I'd rename `Plisteners` to `put_listeners`.

**Review by mairas — COMMENTED** [review]
Thanks for the first contribution and sorry about the delay; I'm on a holiday sail and haven't been able to dig out my computer for quite a few days!

The PR looks great; I had just one minor request on variable naming.

File: src/sensesp/net/ws_client.cpp (hunk 1 above)
Comment (JONA-GA) [reply src/sensesp/net/ws_client.cpp:278]: Nice to have holiday sail !! of course, ok for the variable name, I am not a professional programmer, just a retired biomedical engineer. I will try to read the Google C++ Style Guide. 
Not sure that it's possible to optimize the code by using one vector and one loop (less program memory), I found it more readable like that.

**Comment by mairas** [comment]
Looks great now, thanks!

**Pull request merged**